### PR TITLE
add file comments.js

### PR DESCRIPTION
**Pull request by KostyaKlepach**

### CODE REVIEW

File: client/src/store/modules/comments.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import Vue from "vue"
+import { firebaseDb } from "../../config/config.js"
+
+let messagesRef
+
+const state = {
+  userDetails: {
```
Comment (igorkokotko) [inline client/src/store/modules/comments.js:7]: Why you need hardcoded values here?

File: client/src/store/modules/comments.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  },
+  removeMessage (state, payload) {
+    Vue.delete(state.messages, payload.messageId)
+  },
+  clearMessages (state) {
+    state.messages = {}
+  }
+}
+const actions = {
+  firebaseGetMessages ({ commit, state }, postId) {
```
Comment (igorkokotko) [inline client/src/store/modules/comments.js:27]: Why function with getMessages contains logic with addMessage to store?
Why one action corresponding for addMessage and removeMessage?

File: client/src/store/modules/comments.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    })
+  },
+  firebaseStopGettingMessages ({ commit }) {
+    if (messagesRef) {
+      messagesRef.off('child_added')
+      commit('clearMessages')
+    }
+  },
+  firebaseSendMessage (state, payload) {
+    firebaseDb.ref('/comments').push(payload.message)
```
Comment (igorkokotko) [inline client/src/store/modules/comments.js:51]: Where is the mutation?

File: client/src/store/modules/comments.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  },
+  firebaseDeleteMessage (state, payload) {
+    const messageRef = firebaseDb.ref('/comments/' + payload.id)
```
Comment (igorkokotko) [inline client/src/store/modules/comments.js:54]: Please move all db methods outside of the actions.

File: client/src/store/modules/comments.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    messageRef.remove()
+  }
+}
+const getters = {
+  posts: state => {
```
Comment (igorkokotko) [inline client/src/store/modules/comments.js:59]: if you use arrow function why you need return keyword?

**Review by igorkokotko — CHANGES_REQUESTED** [review]

**Review by igorkokotko — APPROVED** [review]

**Comment by KostyaKlepach** [comment]
todo - remove after demo

**Pull request merged**